news/2024/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments.md

```diff
@@ -0,0 +1,227 @@
+
+I played my first tournament in 2015; it was some 5 digit tournament that I don’t even remember the name of. At one point, I also played in an iteration of SMT, which was a large monthly tournament series back in the day, co-hosted by Halfslashed, who some people reading might still be familiar with. From there I played tournaments off and on, mostly focusing on grinding in solo. I was unable to really focus on tournaments due to my commitment to debate in university; we would travel every 2 – 3 weeks on the weekend, which made committing to a long, multi-week tournament virtually impossible.
+
+Fast forward to the end of university, and I was still craving the same sort of public speaking outlet as debate, but with very few competitions left and a lot of upheaval in the staff team at the time. I decided to try out commentating osu! tournaments, and instantly took to it. The first three tournaments I signed up to staff for were all very good experiences for a first timer, so I was very motivated to continue. Shout out to nik, Tuco, and deflateddolphin for those first three tournaments, as had they been poorly run or overall poor experiences, I almost certainly would not have stuck with it.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/avatar-Utiba.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+As a host of The Perennial tournament series, what makes it stand out to you that makes you want to continue hosting the tournament each year?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3958619" style="background-image: url('/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/avatar-D I O.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>
+
+When I first started it, the Perennial was one of two well-regarded large community 4v4 tournaments in osu! standard. At the start, I focused on a large prize pool and community hype to draw in as many people as possible. Now, several years later, my focus has shifted more to uniqueness in prizing and format while still preserving a similar level of production as previous iterations.
+ 
+The Perennial has never been completely unique, so it instead stands out with the competitiveness of its matches, especially in late stages. In 2021, this was not so much the case, but in 2022 and 2024, we had very competitive grand finals matches between the top teams, with both those grand finals going to bracket reset. In 2024, this was partially due to the format change. Watching these competitive, high-level matches in a tournament you host, on a pool you made, is always very rewarding.
```

Comment:
```suggestion
The Perennial has never been completely unique, so it instead stands out with the competitiveness of its matches, especially in late stages. In 2021, this was not so much the case, but in 2022 and 2024, we had very competitive grand finals matches between the top teams, with both those grand finals going to bracket reset. This was partially due to the format change in 2024. Watching these competitive, high-level matches in a tournament you host, on a pool you made, is always very rewarding.
```

The original version is fine as well, I'm just trying to reduce the usage of commas here